src/libraries/System.IO.Ports/pkg/runtime.native.System.IO.Ports.props

```diff
@@ -17,12 +17,43 @@
     <PackageReadmeFilePath>$(MSBuildThisFileDirectory)../src/PACKAGE.md</PackageReadmeFilePath>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageSupportedRID Include="android-arm" />
+    <PackageSupportedRID Include="android-arm64" />
+    <PackageSupportedRID Include="android-x64" />
+    <PackageSupportedRID Include="android-x86" />
+    <PackageSupportedRID Include="linux-arm" />
+    <PackageSupportedRID Include="linux-arm64" />
+    <PackageSupportedRID Include="linux-bionic-arm64" />
+    <PackageSupportedRID Include="linux-bionic-x64" />
+    <PackageSupportedRID Include="linux-musl-arm" />
+    <PackageSupportedRID Include="linux-musl-arm64" />
+    <PackageSupportedRID Include="linux-musl-x64" />
+    <PackageSupportedRID Include="linux-x64" />
+    <PackageSupportedRID Include="maccatalyst-arm64" />
+    <PackageSupportedRID Include="maccatalyst-x64" />
+    <PackageSupportedRID Include="osx-arm64" />
+    <PackageSupportedRID Include="osx-x64" />
+
+    <!-- We need to add a placeholder file for all RIDs except the one that is actually included in the RID-specific package.
```

Comment:
Are these placeholder files required for all RIDs, or just for the RIDs that are part of a hierarchy (like linux and linux-musl).

Reply:
Strictly required only for conflicting RIDs but I felt it's easier to reason about if we just include it everywhere (they have zero cost) so you don't need to think about whether they're needed or not, e.g. while it might be obvious for linux-musl-x64 -> linux-x64 the relationship between android-x64 -> linux-bionic-x64 is less obvious if you don't know the details.